lib/index.js

```diff
@@ -54,7 +54,7 @@ internals.Connection.prototype.start = function (callback) {
 
     // Wait for connection
 
-    client.once('connect', function () {
```

Comment:
Is there a reason of this change ?
I can read in [node_redis documentation](https://github.com/NodeRedis/node_redis#user-content-connect) that `ready` event is not called if `client.options.no_ready_check` is set.